### ISSUE TITLE
Accessibility - Lighthouse (dev tools)

### ISSUE THREAD

**Issue by nicolegooden** [CalebCyphers/Whats-Cookin#46]
You must score as close to 100% as possible with the “Lighthouse Accessibility Audit” (you’ll need to deploy to GH pages for this). Be prepared to explain any accessibility audits your application is failing.
